index.js

```diff
@@ -24,6 +25,7 @@ const action = async context => {
 	const split = context.config.get('path').split('/');
 	const bucket = split.shift();
 	const filename = path.basename(filePath);
+	const objectKey = path.join(split.join('/'), filename);
```

Comment:
Can you extract `const s3Path = context.config.get('path');` and use that one on line 25 and 28?

Reply:
I'm not sure I fully understand.  If I extract that bit I could use it on line 25 (`const split = s3Path.split('/');`) but I don't think I could use it on line 28 because `s3Path` would include the bucket name.  `objectKey` should be everything bug the bucket name.  If path `config.get('path')` is "mySpecialBucket/kap" `objectKey` should be set to `/kap` since we will prepend with `baseURL` to construct the URL.  Using `split.join('/')` on line 28 seems to this since the array has already had the bucket part removed (`split.shift();` on line 26).

Or, maybe I am not completely following.

Reply:
Oh you are right :)! Missed the shift() method. Looks good then!